### PR TITLE
docs: modify user preferences refactor

### PR DESCRIPTION
**Pull request by chri2547**
This PR refactors user preferences documentation.

### CODE REVIEW

**Review by mjseaman — COMMENTED** [review]
Looks good! I added some extra information.

Notes:
- I think this could live in the Administration section instead of at the very top level of the docs hierarchy.
- There is a Grafana Preferences section at /administration/preferences/ that describes places to set user preferences, but it's hard to discover and understand as a regular user. I opted to repeat some of the preferences information as well as refer to that section.

File: docs/sources/manage-user-preferences/_index.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ Your profile includes your name, user name, and email address, which you can upd
 
 ## Edit your preferences
 
-You can choose the way you would like data to appear in Grafana, including the UI theme, home dashboard, timezone, and first day of the week. You can set these preferences for your own account, for a [team]({{< relref "../administration/manage-users-and-permissions/manage-teams/_index.md">}}), for an organization, or Grafana-wide using [configuration settings]({{< relref "../administration/configuration/_index.md">}}). Your user preferences take precedence over team, organization, and Grafana default preferences.
+You can choose the way you would like data to appear in Grafana, including the UI theme, home dashboard, timezone, and first day of the week. You can set these preferences for your own account, for a team, for an organization, or Grafana-wide using configuration settings. Your user preferences take precedence over team, organization, and Grafana default preferences. For more information, see [Grafana Preferences]({{< relref "../administration/preferences/_index.md">}}).
```
Comment (osg-grafana) [inline docs/sources/manage-user-preferences/_index.md:45]:
```suggestion
You can choose the way you would like data to appear in Grafana, including the UI theme, home dashboard, timezone, and first day of the week. You can set these preferences for your own account, for a team, for an organization, or Grafana-wide using configuration settings. Your user preferences take precedence over team, organization, and Grafana default preferences. For more information, see [Grafana preferences]({{< relref "../administration/preferences/_index.md">}}).
```
Change upstream if needed.

**Review by osg-grafana — COMMENTED** [review]
Use sentence case. For more information, see https://developers.google.com/style/capitalization#capitalization-in-titles-and-headings.

**Review by mjseaman — APPROVED** [review]
LGTM!

**Pull request merged**